data.json

```diff
@@ -223,6 +231,14 @@
         "Jason Sklar", 
         "Randy Sklar"
     ], 
+    "StartUp": [
+	"Alex Blumberg",
```

Comment:
My bad.  I'll fix that and resubmit.



--
Nicholas

On Sat, Nov 28, 2015 at 7:05 AM, Joshua H. Fogg <notifications@github.com>
wrote:

>> @@ -223,6 +231,14 @@
>>          "Jason Sklar", 
>>          "Randy Sklar"
>>      ], 
>> +    "StartUp": [
>> +	"Alex Blumberg",
> Missing space before Alex
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/alexwlchan/hurley/pull/6/files#r46079312